src/MagnumPlugins/ObjImporter/ObjImporter.cpp

```diff
@@ -28,81 +28,338 @@
 #include <fstream>
 #include <limits>
 #include <sstream>
+#include <tuple>
+#include <stdlib.h>
```

Comment:
I think that was because of [strtol](http://www.cplusplus.com/reference/cstdlib/strtol/) and strtof